students/TracyA/session11/featuresdf.csv

```diff
@@ -0,0 +1,101 @@
+id,name,artists,danceability,energy,key,loudness,mode,speechiness,acousticness,instrumentalness,liveness,valence,tempo,duration_ms,time_signature
```

Comment:
try and get in the habit of creating a .gitignore file where you include *.csv or other dataset types. Basically, you don;t want to upload data onto github if you can help it.